pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -773,6 +773,9 @@ type SeedMLASettings struct {
 // MeteringConfiguration contains all the configuration for the metering tool.
 type MeteringConfiguration struct {
 	Enabled bool `json:"enabled"`
+
+	// +kubebuilder:default=kubermatic-fast
```

Comment:
Let's not hardcode this value into our CRDs. If anything we should do the defaulting in our own code and make this maybe dependent on the KubermaticConfiguration. "kubermatic-fast" was also originally only meant for SSD volumes for etcds, not for "everthing that KKP stores", so `kubermatic-fast` might even be the wrong default here.

I would just use the default StorageClass from the seed cluster, unless the admin configures a special SC in the Seed datacenter.